documentation/modules/deploying/proc-deploy-kafka-bridge.adoc

```diff
@@ -32,20 +32,22 @@ kubectl apply -f examples/bridge/kafka-bridge.yaml
 +
 [source,shell,subs="+quotes"]
 ----
-kubectl get deployments -n _<my_cluster_operator_namespace>_
+kubectl get pods -n _<my_cluster_operator_namespace>_
 ----
 +
 .Output shows the deployment name and readiness
 [source,shell,subs="+quotes"]
 ----
-NAME              READY  UP-TO-DATE  AVAILABLE
-my-bridge-bridge  1/1    1           1
+NAME              READY  STATUS   RESTARTS
+my-bridge-bridge  1/1    Running  0
```

Comment:
The name here would be more like `my-bridge-bridge-<xxxxxxxx>-<xxxxx>`. Thsi would apply also to the Connect and Mirror MAker 2 pods later when the feature gate is not enabled. So I wonder if we want to use something like `<id> everywhere for the suffix.

Reply:
Yes. I thought to leave it out.  Is something missing from the comment? We could use something like `<pod_id>` that applies to sequential and random? Or I can say something about both for the connect/mm2 steps and we keep the sequential in the example?